Сетка страниц и крупных блоков на флексах

Ierixorn opened the pull request:
Выравнивание основных блоков страниц с использованием flex.

---
:mortar_board: [Сетка страниц и крупных блоков на флексах](https://up.htmlacademy.ru/htmlcss/33/user/1898375/tasks/7)

:boom: https://htmlacademy-htmlcss.github.io/1898375-gllacy-33/6/

Ierixorn committed:
Настройка глобальной сетки

Выравнивание и расстановка основных блоков сайта

Ierixorn committed:
Завершение сетки основных блоков обеих страниц

Добавление сетки основных блоков на главную страницу и каталог

sealeye commented on css/style.css:
```diff
@@ -22,13 +22,22 @@
+  display: flex;
+  flex-direction: column;
+  margin: 0;
 	font-family: "Inter", sans-serif;
```
Нужно сохранять единообразие отступов

sealeye commented on css/style.css:
```diff
@@ -22,13 +22,22 @@
 	background: #FEAFC3;
 }
 
+.blobal-layout {
```
Странное название для класса

sealeye commented on site_pages/catalog.html:
```diff
@@ -9,13 +9,15 @@
-      </a>
       <ul class="navigation-links global-list">
+        <li class="navigation-item">
+          <a class="navigation-logo">
```
Это не часть списка, лого должно идти отдельно. 
Потому что внутри списка все элементы должны выглядеть одинаково и иметь одинаковую функцию. Лого выбивается из этого ряда

sealeye commented:
У тебя очень хорошо получилась сетка, ты строишь её достаточно логично. 
Но так как мы начинаем работать с визуалом, то нужно чтобы проект хорошо публиковался на GitHub pages, для этого нужно удалить папку site_pages и положить файлы разметки в корневую папку проекта. 
Тем более что в работе с реальными серверами придётся делать также.

Ierixorn committed:
Исправление ошибок (4 домашнее задание)

-Перенос страниц в "корень" репозитория.
- Перенос лого магазина в шапке из списка.
- Мелкие исправления стилевого файла.
